R/table-subscript.R

```diff
@@ -312,6 +313,45 @@ updateIsSubscriptedAttr <- function(y, x) {
     y
 }
 
+updateCellText <- function(y, x.attributes, evaluated.args) {
+    cell.text <- x.attributes$celltext
+    if (!is.array(cell.text)) {
+        return (y)
+    }
+
+    # convert row/column/stat name args to numeric indices
+    indices <- lapply(seq_along(evaluated.args), function(i) {
+        if (is.character(evaluated.args[[i]]) && !is.null(x.attributes$dimnames)) {
+            dim.names <- x.attributes$dimnames[[i]]
+            all.indices <- seq_along(dim.names)
+            names(all.indices) <- dim.names
+            all.indices[evaluated.args[[i]]]
+        } else {
+            evaluated.args[[i]]
+        }
+    })
+
+    subscripted <- do.call(`[`, c(list(cell.text), indices))
+
+    if (!is.array(subscripted)) {
+        subscripted <- array(subscripted)
+    }
+
+    attr(y, "celltext") <- subscripted
+
+    y
+}
+
+flattenIndices <- function(indices.1, indices.2, dim.1, dim.2) {
+    if (isEmptyArg(indices.1)) {
+        indices.1 <- seq_len(dim.1)
+    }
+    if (isEmptyArg(indices.2)) {
+        indices.2 <- seq_len(dim.2)
+    }
+    rep(indices.1, length(indices.2)) + (indices.2 - 1) * dim.1
+}
```

Comment:
Is this function used? Could be deleted if not?

Reply:
Nope, I'll remove